service-post/src/main/java/com/team13/servicepost/controller/PostController.java

```diff
@@ -36,53 +38,69 @@ public ResponseEntity<PostResponseDto> createPost(
         }
     }
 
-    @GetMapping("/{id}")
-    public ResponseEntity<PostResponseDto> getPostById(@PathVariable("id") Long id) {
-        Optional<PostResponseDto> postWithDetails = postService.getPostWithUserDetails(id);
+    @GetMapping("/{postId}")
+    public ResponseEntity<PostResponseDto> getPostById(@PathVariable("postId") Long postId) {
+        Optional<PostResponseDto> postWithDetails = postService.getPostWithUserDetails(postId);
         if (postWithDetails.isPresent()) {
             return ResponseEntity.ok(postWithDetails.get());
         } else {
             return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
         }
     }
 
-    // localhost:925:1/like?userId=2
-    //userId가 2인 사람이 Post1번 글 좋아요를 누른다.
-    @PostMapping("/{postId}/like")
-    public ResponseEntity<String> toggleLikePost(@PathVariable("postId") Long postId, @RequestParam Long userId) {
-        boolean success = likePostService.toggleLikePost(postId, userId);
-        if (success) {
-            return ResponseEntity.ok("게시글에 대한 좋아요 혹은 좋아요 취소가 실행됐습니다");
-        } else {
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("해당 게시글 혹은 유저확인이 문제로 좋아요 관련 기능이 실행되지 않았습니다.");
+    @PostMapping("/like/{postId}")
+    public ResponseEntity<String> toggleLikePost(@PathVariable("postId") Long postId, HttpServletRequest request) {
+        String userIdHeader = request.getHeader("X-User-Id");
+        try {
+            Long userId = Long.valueOf(userIdHeader);
+            boolean success = likePostService.toggleLikePost(postId, userId);
+            if (success) {
+                return ResponseEntity.ok("게시글에 대한 좋아요 혹은 좋아요 취소가 실행됐습니다");
+            } else {
+                return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("해당 게시글 혹은 유저확인이 문제로 좋아요 관련 기능이 실행되지 않았습니다.");
+            }
```

Comment:
```suggestion
            return likePostService.toggleLikePost(postId, userId)
                ? ResponseEntity.ok("게시글에 대한 좋아요 혹은 좋아요 취소가 실행됐습니다")
                : ResponseEntity.status(HttpStatus.BAD_REQUEST).body("해당 게시글 혹은 유저확인이 문제로 좋아요 관련 기능이 실행되지 않았습니다.");
```

위와 같이 3항 연산자를 사용하면 코드가 좀 더 깔끔할 것 같습니다!

Reply:
넵 좋습니다! 더 코드가 깔끔하네요~!